db/migrate/20171113153142_create_instructors.rb

```diff
@@ -0,0 +1,9 @@
+class CreateInstructors < ActiveRecord::Migration[5.1]
+  def change
+    create_table :instructors do |t|
+    	t.string :name
+    	t.string :email
+      t.timestamps
```

Comment:
There is some weird indentation going on here 😅

Reply:
Also, name and email should have database null-restrictions, to ensure that our database has its own protections against invalid data.
```
  t.string :name, null: false
  t.string :email, null: false
```

Reply:
Regarding the indentation, there is no problem in my local document...weird

Reply:
![Uploading Screenshot 2017-11-20 22.01.42.png…](Image)